src/base/OpaqueId.hh

```diff
@@ -142,6 +142,5 @@ struct hash<celeritas::OpaqueId<I, T>>
     }
 };
 } // namespace std
-#endif
-
 //---------------------------------------------------------------------------//
```

Comment:
Out of curiosity, what is the (visual) semantic of this trailing comment/line?

Reply:
It's meant to be a visual separator of the interior of the `#ifdef __CUDAARCH__`... but it's not very effective here. The weird bit is whether to put dividers on the inside or outside of `#if/#endif` or `{/}` blocks, and then if you're subdividing within those sections, where to put them for consistency...

Do you have a suggestion here? And I'm assuming the suggestion won't be to replace with the Geant4 separators
```
//....oooOO0OOooo........oooOO0OOooo........oooOO0OOooo........oooOO0OOooo......
```
👻👻👻

Reply:
I would put separator (which ever spelling there of :) ) only at the start

Reply:
I've done this and also simplified and unified numerous other minor formatting issues in a separate merge request #61